src/roblox_api/instance.rs

```diff
@@ -91,6 +99,119 @@ impl LuaInstance {
         Ok(child)
     }
 
+    fn get_attribute<'lua>(
+        &self,
+        context: rlua::Context<'lua>,
+        attribute_name: &str,
+    ) -> rlua::Result<Option<rlua::Value<'lua>>> {
+        let tree = self.tree.lock().unwrap();
+
+        let instance = tree.get_by_ref(self.id).ok_or_else(|| {
+            rlua::Error::external("Cannot call GetAttribute() on a destroyed instance")
+        })?;
+
+        let attribute_binary_string = match instance.properties.get("AttributesSerialize") {
```

Comment:
As a heads-up, we'll probably be tweaking the reflection database before attributes go live so that this property becomes just `Attributes` when in memory.

Reply:
Attributes have been live for a few weeks now 😛

Reply:
*before attributes go live in rbx-dom :)